packs/journals/archetypes.json

```diff
@@ -1263,7 +1263,7 @@
+                "content": "<p>You're a skilled performer, with impressive balance and coordination that help you greatly in battle.</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Juggler Dedication] <span style=\"float:right\">Feat 2</span></h2><p><strong>Prerequisites</strong> Dexterity +2</p><hr /><p>You become trained in Performance; if you were already trained in Performance, you instead become trained in a skill of your choice. You also gain the @UUID[Compendium.pf2e.feats-srd.Item.Juggle] skill feat, even if you don't meet its prerequisites.</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Focused Juggler] <span style=\"float:right\">Feat 4</span></h2><div><p>@Embed[Compendium.pf2e.feats-srd.Item.lYVAGuHU47Ixyuxy inline hr=false]</p></div><h2>@UUID[Compendium.pf2e.feats-srd.Item.Quick Juggler] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.otLs7XzMXR1cZKGe inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Reflexive Catch] <span style=\"float:right\">Feat 4</span></h2><div><p>@Embed[Compendium.pf2e.feats-srd.Item.irDFSzKCaF4ux3bx inline hr=false]</p></div><h2>@UUID[Compendium.pf2e.feats-srd.Item.Lobbed Attack] <span style=\"float:right\">Feat 8</span></h2><div><p>@Embed[Compendium.pf2e.feats-srd.Item.XkmrLhyAoxQTLnza inline hr=false]</p></div><h2>@UUID[Compendium.pf2e.feats-srd.Item.Opportune Throw] <span style=\"float:right\">Feat 16</span></h2><div><p>@Embed[Compendium.pf2e.feats-srd.Item.2LdncNMDNJW5Oeyu inline hr=false]</p></div>",
```

Comment:
This one ends with a `</div>` tag, but the other's don't. Is this intentional?

Reply:
Should be fixed now

Reply:
The opening div tag is still present, and there are numerous others in this entry.